apicast/cpanfile

```diff
@@ -0,0 +1 @@
+requires 'Test::APIcast', '0.04';
```

Comment:
`0.04` is an old version. Although the latest will be installed anyway unless we pin the version with `==`.

Reply:
Yep. I think that is fine. This library is already installed in the apicast image so this just ensures it has the right version.